src/scss/vault.scss

```diff
@@ -29,274 +28,6 @@ app-root {
 
   > .items {
     order: 2;
-  }
```

Comment:
I love breaking up these stylesheets, yessir

Reply:
Still a lot to do here unfortunately, even the `left-nav` file I made probably needs further breaking down.